mpi-jobs.shtml

```diff
@@ -149,11 +170,16 @@ a submit file for multicore jobs.  They are shown in the sample submit
 file below and include: </p>
 
 <ul>
-	<li><b>Require access to MPI modules.</b> There is a special requirements statement
-	that ensures a computer has access to Gluster (where the MPI modules live) and 
-	that the module command is working. </li>
+	<li><b>Require access to MPI modules.</b> To ensure that your job will have access 
+	to CHTC software modules, including MPI modules, you must include the following in your submit file. 
+	<pre class="sub">
+requirements = (HasCHTCSoftware == true)
```

Comment:
This attribute is called "HasChtcSoftware" (case difference) in the example job below.  Match the two?